package.json

```diff
@@ -19,7 +19,7 @@
     "utility"
   ],
   "dependencies": {
-    "lodash": "^4.14.0",
+    "lodash": "^4.17.5",
     "lodash-es": "^4.14.0"
```

Comment:
Does `lodash-es` need to be updated too?

Reply:
`lodash-es` doesn't depend on `lodash`, it's fine as is

Reply:
As far as I know, `lodash-es` is built from `lodash`. 

Either way, it's probably a good idea to keep them synced. Could you update it to `v4.17.5`? The PR will be good to go afterwards.

Reply:
You are right! Done!